xml/templates/message_templates/contribution_online_receipt_html.tpl

```diff
@@ -311,7 +311,7 @@
       </tr>
      {/if}
 
-     {if ! ($contributeMode eq 'notify' OR $contributeMode eq 'directIPN') and $is_monetary}
+     {if ($is_pay_later || $billingName)}
```

Comment:
If you remove this file's changes for later the tests won't fail & we can do the others